Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -32,13 +32,37 @@ struct PitchSpeller {
             UnorderedPair(.down,.down)
         ]
     )
-
-    static let adjacencyCarrying = AdjacencyCarrying.build(from: PitchSpeller.tendencyGraph)
-    static let tendencyMask: AdjacencyCarrying<Graph<FlowNode<Cross<Int,Tendency>>>> = adjacencyCarrying.pullback { node in
+    
+    static let eightTendencyLink: [(Pitch.Class, Tendency)] = [
+        (.init(value: 0), .down),
+        (.init(value: 1), .up),
+        (.init(value: 3), .down),
+        (.init(value: 4), .up),
+        (.init(value: 5), .down),
+        (.init(value: 6), .up),
+        (.init(value: 7), .down),
+        (.init(value: 8), .up),
+        (.init(value: 9), .up),
+        (.init(value: 10), .down),
+        (.init(value: 11), .up),
+        ]
```

Comment:
Bring this is one level.

Reply:
Wait, could you clarify what you mean by this, @jsbean  ??

Reply:
Ah sorry, syntax nit: bring in the `]` one level of indentation.

Reply:
Oh, yes, definitely!